libr/bin/p/bin_mach0.c

```diff
@@ -358,16 +358,14 @@ static RBinImport *import_from_name(RBin *rbin, const char *orig_name, HtPP *imp
 
 	char *name = (char*) orig_name;
 	const char *_objc_class = "_OBJC_CLASS_$";
-	const int _objc_class_len = strlen (_objc_class);
 	const char *_objc_metaclass = "_OBJC_METACLASS_$";
-	const int _objc_metaclass_len = strlen (_objc_metaclass);
 	const char * type = "FUNC";
 
-	if (!strncmp (name, _objc_class, _objc_class_len)) {
-		name += _objc_class_len;
+	if (r_str_startswith (name, _objc_class)) {
```

Comment:
uh, this seems dupe  :D so do the same as in mach0.c 3280

Reply:
Yeah, that code is used by `patch_relocs`, which I haven't touched yet. Will probably also be moved over to the other file once we cache all that..